src/display.rs

```diff
@@ -312,8 +317,11 @@ impl Display {
                 item.on_resize(size)
             }
 
+            let cw = self.size_info.cell_width as i32;
```

Comment:
Why are you casting here and converting other places?

Reply:
I can only use the `From` trait when implemented, that is not the case for every float/integer conversion.